frog/imports/ui/Home.jsx

```diff
@@ -0,0 +1,80 @@
+import React, { Component, PropTypes } from 'react';
+import ReactDOM from 'react-dom';
+import { Meteor } from 'meteor/meteor';
+import { createContainer } from 'meteor/react-meteor-data';
+import { Messages } from '../api/messages.js';
+
+export default class Home extends Component {
+
+  renderUsers(){
+    return (
+      this.props.presentUsers ? 
+      this.props.presentUsers
+        .map((user)=> user.userId)
+        .map((userId) =>
+          <li key={userId}>{Meteor.users.findOne({_id: userId}).username}</li>
```

Comment:
When a user opens two windows, it's id will be twice in the list. The display is correct as the `<li></li>`will have the same key, so only one will be shown. However it raises a warning (see image), which we want to avoid. You can easily filter your list to keep only unique userId.
![screen shot 2016-10-13 at 15 37 15](https://cloud.githubusercontent.com/assets/8818081/19351179/018bd00a-915b-11e6-80e8-12cfc46f5dd5.png)